src/Database/QueryCompiler.php

```diff
@@ -85,6 +85,13 @@ class QueryCompiler
      */
     protected $_orderedUnion = true;
 
+    /**
+     * Indicate whether identifiers of SELECT clause need to be always quoted.
+     *
+     * @var bool
+     */
+    protected $_quotedSelectIdentifiers = false;
```

Comment:
This is specifically for quoting aliases right, not column identifiers.

Reply:
Yes, should I rename the property to better reflect that?

Reply:
I think so.